airflow/providers/amazon/aws/triggers/redshift_cluster.py

```diff
@@ -70,6 +70,16 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:
                     else:
                         if self.attempts < 1:
                             yield TriggerEvent({"status": "error", "message": f"{self.task_id} failed"})
+                elif self.operation_type == "resume_cluster":
+                    response = await hook.resume_cluster(
+                        cluster_identifier=self.cluster_identifier,
+                        polling_period_seconds=self.poll_interval,
+                    )
+                    if response.get("status") == "success":
+                        yield TriggerEvent(response)
+                    else:
+                        if self.attempts < 1:
+                            yield TriggerEvent({"status": "error", "message": f"{self.task_id} failed"})
```

Comment:
```suggestion
                if response:
                    yield TriggerEvent(response)
                else:
                    error_message = f"{self.task_id} failed"
                    yield TriggerEvent({"status": "error", "message": error_message})
```

Reply:
Why is `self.attempts` not checked here?

Reply:
Absolutely ! @o-nikolas - I've been looking at it before our provider's releases and I hope @eladkal did too.

BTW. Today I was using it as an example for the Databricks team that is looking to build similar dashboard for databricks provider (cc: @alexott ) - I think there might be a learning or two to share between your teams :)

Reply:
> BTW. Today I was using it as an example for the Databricks team that is looking to build similar dashboard for databricks provider (cc: @alexott ) - I think there might be a learning or two to share between your teams :)

Exciting! Feel free to reach out @alexott if you have any questions or would like to brainstorm :smiley: